draft-ietf-taps-transport-security.md

```diff
@@ -383,7 +376,9 @@ IKEv2 {{RFC7296}} and ESP {{RFC4303}} together form the modern IPsec protocol su
 
 #### IKEv2
 
-IKEv2 is a control protocol that runs on UDP port 500. Its primary goal is to generate keys for Security Associations (SAs). It first uses a Diffie-Hellman key exchange to generate keys for the "IKE SA", which is a set of keys used to encrypt further IKEv2 messages. It then goes through a phase of authentication in which both peers present blobs signed by a shared secret or private key, after which another set of keys is derived, referred to as the "Child SA". These Child SA keys are used by ESP.
+IKEv2 is a control protocol that runs on UDP port 500. Its primary goal is to generate keys for Security Associations (SAs). 
+A SA contains shared (cryptographic) information used for establishing other SAs or keying ESP; See {{ESP}}.
```

Comment:
"An" SA

Reply:
Hmm... I read SA out loud, not as an abbreviation. But this is a nit so I'll make the change.